UnitsNet/IQuantity.cs

```diff
@@ -16,6 +16,7 @@ public interface IQuantity : IFormattable
         /// <summary>
         ///     The <see cref="QuantityType" /> of this quantity.
         /// </summary>
+        [Obsolete("QuantityType will be removed in the future. Use QuantityInfo instead.")]
```

Comment:
I see the [AppVeyor build failed](https://ci.appveyor.com/project/angularsen/unitsnet/builds/36676346) on obsolete errors. We probably need to add this to `UnitsNet.Tests.csproj`

```
<WarningsNotAsErrors>612,618</WarningsNotAsErrors>
```

Even better, put it in `Directory.Build.props` so it affects all projects that have `<MSBuildTreatWarningsAsErrors>true</MSBuildTreatWarningsAsErrors>` in them.

Or use pragma statements as you already have some places.

Reply:
done, works like a charm. Restart Visual Studio though.